libmamba/ext/solv-cpp/tests/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ add_executable(
     src/main.cpp
     src/msvc_catch_string_view.cpp
     src/pool_data.cpp
+    src/pool_data.hpp
```

Comment:
There is no separation for source and header files here, so we should mention this header file as well.

Reply:
Yes.
And it also makes it consistent (all source/header files are present in CMake files).